CITATION.cff

```diff
@@ -0,0 +1,35 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+title: "Triton Inference Server: an optimized cloud and edge inferencing solution."
```

Comment:
The subtext needs to be in proper title format. "An Optimized Cloud and Edge Inference Solution"